packages/documentation/src/content/docs/integration/requirements/wallet-addresses.mdx

```diff
@@ -194,11 +193,11 @@ Open Payments <LinkOut href="https://openpayments.dev/apis/wallet-address-server
 
 <div class="overflow-table">
 
-| Parameter | Required value | Description                                                                   |
```

Comment:
was this auto-formatted?

Reply:
When we run prettier on the docs, it adds a bunch of extra spaces within all the markdown table columns and rows. That makes it harder for the writers to 'read' the tables and edit them later. Sarah's given us a temporary workaround where prettier will ignore the tables (actually, maybe all the doc files) and keep our original formatting. So this was me putting the table back to its original formatting.